### ISSUE TITLE
Output directory in _get_jdk_module_jar doesn't work on Windows

### ISSUE THREAD

**Issue by harmanpa** [graalvm/mx#148]
The `jdkOutputDir` path in `_get_jdk_module_jar` is determined using the `jdk.home` property, with the leading slash removed:
https://github.com/graalvm/mx/blob/d14e55ab0f64d0384dc3c8fe27eb96c2408d3352/mx.py#L13246

However on Windows this doesn't work as there is the drive letter also to remove, the following works though there may be a neater implementation:

    if get_os() == 'windows':
        jdkOutputDir = ensure_dir_exists(join(suite.get_output_root(), os.path.abspath(jdk.home)[3:]))
    else:
        jdkOutputDir = ensure_dir_exists(join(suite.get_output_root(), os.path.abspath(jdk.home)[1:]))

**Comment by dougxc** [comment]
can you please let me know if the following works (I don't have access to Windows):
```
    suffix = os.path.splitdrive(os.path.abspath(jdk.home))[1][1:]
    jdkOutputDir = ensure_dir_exists(join(suite.get_output_root(), suffix))
```

**Comment by harmanpa** [comment]
Yes that works, thank you

**Comment by dougxc** [comment]
This fix in now merged: daa2439cd23cec3b3b7050ea30070bb5e57112ea
